model/schema/phenotypeAndDiseaseAnnotation.yaml

```diff
@@ -301,15 +302,16 @@ classes:
       The environmental context in which an experiment is carried out. This may (e.g. drug treatment)
       or may not (e.g. standard conditions) directly influence the outcome of the experiment.
     slots:
-      - unique_id
-      - condition_class                   # conditionClassId in JSON schema
-      - condition_statement               # conditionStatement in JSON schema
-      - condition_id                      # conditionId in JSON schema
-      - condition_quantity                # conditionQuantity in JSON schema
-      - condition_anatomy                 # anatomicalOntologyId in JSON schema
-      - condition_gene_ontology           # geneOntologyId in JSON schema
-      - condition_taxon                   # NCBITaxonId in JSON schema
-      - condition_chemical                # ChemicalOntologyId in JSON schema
+      - unique_id                         # calculated centrally at the Alliance after import from MODs
+      - condition_class                   # conditionClassId in frozen JSON schema
+      - condition_summary                 # conditionStatement in frozen JSON schema; for new DQM submissions should not be submitted but rather calculated centrally at the Alliance
```

Comment:
Tests are currently failing because they're expecting "condition_summary" (required), but test files still have "condition_statement". Fixing files depends on resolution of the above question.